src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -34,37 +35,36 @@ String create(String uid) throws Exception, MeiliSearchApiException {
      * @param uid Unique identifier to create the index with
      * @param primaryKey Field to use as the primary key for documents in that index
      * @return Meilisearch API response
-     * @throws Exception if an error occurs
+     * @throws MeilisearchException if an error occurs
      */
-    String create(String uid, String primaryKey) throws Exception, MeiliSearchApiException {
-        JsonObject params = new JsonObject();
-        params.addProperty("uid", uid);
-        if (primaryKey != null) {
-            params.addProperty("primaryKey", primaryKey);
-        }
-        return meilisearchHttpRequest.post("/indexes", params.toString());
+    Task create(String uid, String primaryKey) throws MeilisearchException {
+        HashMap<String, Object> index = new HashMap<String, Object>();
```

Comment:
Do you think it is possible to specify the String type here? Since both params are strings.
```suggestion
        HashMap<String, String> index = new HashMap<String, String>();
```